docs/plugins/plugins.d.ts

```diff
@@ -0,0 +1,41 @@
+/* tslint:disable */
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
+export interface AspectCLIPluginsCatalog {
+  $schema?: string;
+  plugins: {
+    /**
+     * GitHub repository
+     */
+    "github-repo": string;
```

Comment:
On the registry I think they use `repository`.

Reply:
yeah, but they support more than just github, and we don't yet. Hm. I was going to just barrel ahead but I think you're right, it's better to have `repository = github:some/repo` and then do the error handling/reporting